examples/credential_issuer_metadata_sd_jwt_long.json

```diff
@@ -0,0 +1,133 @@
+{
+  "credential_issuer": "https://credential-issuer.example.com",
+  "authorization_servers": [
+    "https://server.example.com"
+  ],
+  "credential_endpoint": "https://credential-issuer.example.com",
+  "nonce_endpoint": "https://credential-issuer.example.com/nonce",
+  "deferred_credential_endpoint": "https://credential-issuer.example.com/deferred_credential",
+  "notification_endpoint": "https://credential-issuer.example.com/notification",
+  "credential_request_encryption": {
+    "jwks": [
+      {
+        "kty":"EC", "kid":"ac", "use":"enc", "crv":"P-256","alg":"ECDH-ES",
+        "x":"YO4epjifD-KWeq1sL2tNmm36BhXnkJ0He-WqMYrp9Fk",
+        "y":"Hekpm0zfK7C-YccH5iBjcIXgf6YdUvNUac_0At55Okk"
+      }
+    ],
+    "enc_values_supported": ["A128GCM"],
```

Comment:
It'd be legal to omit this as it defaults to A128GCM? But obviously fine to include it too.